generators/client/templates/angular/src/main/webapp/app/layouts/navbar/navbar.scss.ejs

```diff
@@ -42,21 +45,25 @@ Navbar
         background-color: $dark;
     }
 
+    .dropdown-item:hover {
+        color: $dropdown-link-hover-color;
```

Comment:
If I understood this correctly, then, you are trying to revert to default hover color. If yes, then we override that in style above this to align with custom Jhipster theme. If you don't override styles as per my last comment, then, this wouldn't be required.

Reply:
Not it's because bootstrap apply the hover-color only on `a` elements and not on `dropdown-items`. We use sometimes dropdown-items without links and we need to have the same color on hover.

Reply:
I don't see this in navbar template. Are you talking about customization that you do post generation?

Reply:
No it is in the bootstrap code. They apply this color only for the `a` tags inside `dropdown-item`, however we want to apply this color on all `dropdown-item`, whether they contains a link or not.

Reply:
if this change is not scoped to `navbar`, then, shouldn't it go as global style? Also, this should be conditional and not applied if theme is selected to ensure that their default is respected

Reply:
I can move it into global yes.
For me it's a `bootstrap` strange behaviour so I would always apply this fix even with a theme. We use a variable so the color is not hard-coded.